https://issues.redhat.com/browse/ACM-7716 Update customize_observability.adoc (#6070)
open-cluster-management/rhacm-docs: observability/customize_observability.adoc

[#customizing-observability]
= Customizing observability configuration

Customize the observability configuration to the specific needs of your environment, after you enable observability.

> Read the following sections to learn more about how you want to manage and view cluster fleet data that the observability service collects:

Reviewer: Not the biggest deal, but IBM SG encourages us to lead with reason, then action. So this would be:

To learn more about how you want to manage and view cluster fleet data that the observability service collects, read the following sections:

> Read the following sections to learn more about how you want to manage and view cluster fleet data that the observability service collects:

Author: ```suggestion
To learn more about how you want to manage and view cluster fleet data that the observability service collects, read the following sections:
```

Create custom rules for the observability installation by adding Prometheus link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/[recording rules] and link:https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[alerting rules] to the observability resource.

> Recording rules give you the ability to precalculate expensive expressions as needed. The results are saved as a new set of time series. Alerting rules give you the ability to specify the alert conditions based on how you want to send an alert to an external service.

Reviewer: To avoid anthropomorphism and to make it more user focused, we could reword this first sentence to something like:

To precalculate expensive expressions, use the recording rules abilities. The results are saved as a new set of time series. With the alerting rules, you can specify the alert conditions based on how you want to send an alert to an external service.

> Recording rules give you the ability to precalculate expensive expressions as needed. The results are saved as a new set of time series. Alerting rules give you the ability to specify the alert conditions based on how you want to send an alert to an external service.

Author: ```suggestion
To precalculate expensive expressions, use the recording rules abilities. The results are saved as a new set of time series. With the alerting rules, you can specify the alert conditions based on how you want to send an alert to an external service.
```
